website/src/community/twitter-handle.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: section
+title: 'Beam Twitter handle'
+section_menu: section-menu/community.html
+permalink: /community/twitter-handle/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Beam Twitter handle 
+## What is it and what are the goals?
+Apache Beam community thrives to be inclusive to everyone. As part of this effort, we want to enable any community member to share anything interesting and fun related to Beam via its [official Twitter handle](https://twitter.com/ApacheBeam).
```

Comment:
@aijamalnk Thanks for helping out on this - really glad this is part of the webpage now.

Am I misunderstanding this sentence or do you mean 'Apache Beam community tries...' or 'Apache Beam community thrives by being...'? :)

Reply:
@matthiasa4 it is tries :) feel free to submit a PR with the fix :)